tests/core/test_rpc_util.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import List
+
+import pytest
+
+from chia.rpc.util import RequestType, marshall
+from chia.util.ints import uint32
+from chia.util.streamable import Streamable, streamable
+
+
+@streamable
+@dataclass(frozen=True)
+class SubObject(Streamable):
+    qux: str
+
+
+class TestRequestType(RequestType):
+    foofoo: str
+    barbar: uint32
+    bat: bytes
+    bam: SubObject
+
+
+@streamable
+@dataclass(frozen=True)
+class TestResponseObject(Streamable):
+    qat: List[str]
```

Comment:
i think a little more content here would be good to show it can handle bytes and nested types and whatever sensibly.  not 'everything', but more than just list and string which could as well be handled by a typed dict converted to json.

Reply:
Yeah, I wasn't looking to test streamable -> json, just enough to show that this does in fact use that path, kinda.  Thanks.

Reply:
Feel free to close when you've read.